cmd/context/create.go

```diff
@@ -132,3 +150,63 @@ func UseContext(ctx context.Context, ctxOptions *ContextOptions) error {
 
 	return nil
 }
+
+func (c *ContextUse) initOktetoContext(ctx context.Context, ctxOptions *ContextOptions) error {
+	user, err := c.loginController.AuthenticateToOktetoCluster(ctx, ctxOptions.Context, ctxOptions.Token)
+	if err != nil {
+		return err
+	}
+	ctxOptions.Token = user.Token
+	okteto.Context().Token = user.Token
+
+	userContext, err := c.getUserContext(ctx)
+	if err != nil {
+		return err
+	}
+	if ctxOptions.Namespace == "" {
+		ctxOptions.Namespace = userContext.User.Namespace
+	}
+	okteto.AddOktetoContext(ctxOptions.Context, &userContext.User, ctxOptions.Namespace)
+	cfg := kubeconfig.Get(config.GetKubeconfigPath())
+	if cfg == nil {
+		cfg = kubeconfig.Create()
+	}
+	okteto.AddOktetoCredentialsToCfg(cfg, &userContext.Credentials, ctxOptions.Namespace, userContext.User.ID, okteto.Context().Name)
+	okteto.Context().Cfg = cfg
+	okteto.Context().IsOkteto = true
+
+	setSecrets(userContext.Secrets)
+
+	os.Setenv("OKTETO_USERNAME", okteto.Context().Username)
+
+	return nil
+}
+
+func (c *ContextUse) initKubernetesContext(ctxOptions *ContextOptions) error {
```

Comment:
I think `c` is not being used

```suggestion
func (_ *ContextUse) initKubernetesContext(ctxOptions *ContextOptions) error {
```